python/cudf_polars/cudf_polars/experimental/groupby.py

```diff
@@ -54,6 +45,8 @@ def combine(
     -------
     Unified groupby-aggregation decomposition.
     """
+    if len(decompositions) == 0:
```

Comment:
https://github.com/rapidsai/cudf/pull/18277 includes some tests that should exercise this. I can update that once this is merged.